src/MagicOnion.Client.Unity/Assets/Scripts/gRPC/Core/Channel.cs

```diff
@@ -269,11 +269,13 @@ public IObservable<Unit> ShutdownAsync()
             var activeCallCount = activeCallCounter.Count;
             if (activeCallCount > 0)
             {
+#if !UNITY_METRO
                 // wait for shutdown complete...!
                 Logger.Debug("Wait shutdown start 0.5 sec...");
                 System.Threading.Thread.Sleep(TimeSpan.FromMilliseconds(500));
 
                 if (activeCallCounter.Count > 0)
+#endif
```

Comment:
Because `activeCallCount` and `activeCallCounter.Count` are equal, I think there is no need to compare them.